src/nft/components/bag/profile/ListingButton.tsx

```diff
@@ -71,13 +80,27 @@ export const ListingButton = ({ onClick, buttonText, showWarningOverride = false
         for (const listing of asset.newListings) {
           if (!listing.price) listingsMissingPrice.push([asset, listing])
           else if (isNaN(listing.price) || listing.price < 0) invalidPrices.push([asset, listing])
-          else if (listing.price < (asset?.floorPrice ?? 0) && !listing.overrideFloorPrice)
+          else if (listing.price < (asset?.floorPrice ?? 0) * 0.8 && !listing.overrideFloorPrice)
```

Comment:
should we make 0.8 a const here? its a little confusing as to exactly what it does